wordle-web/src/components/AppBar.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <v-app-bar id="toolbar">
+    <v-toolbar-title>
+      <div @click="router.push('/')" id="toolbar-title">
+        <v-icon icon="mdi-book-open-page-variant" />
+        The Good Word
+      </div>
+    </v-toolbar-title>
+
+    <v-btn @click="toggleMusic">
+      <v-icon size="large">{{ isPaused ? 'mdi-volume-high' : 'mdi-volume-mute' }}</v-icon>
+    </v-btn>
+
+    <v-btn icon="mdi-cog" @click.stop="settings = !settings"></v-btn>
+
+    <v-app-bar-nav-icon @click.stop="menu = !menu"></v-app-bar-nav-icon>
+  </v-app-bar>
+
+  <v-dialog transition="fab-transition" width="350px" title="Settings" v-model="settings" temporary>
```

Comment:
```suggestion
  <v-dialog transition="fab-transition" width="350px" title="Settings" v-model="settings" temporary>
```
I learned something new, I didn't know you could do these transitions on dialogs. :)